# This is a gimbal control message
float64 pitch
float64 yaw